packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -231,22 +249,29 @@ class EnumerationWidget extends React.Component {
 		}
 	}
 
+	valueAlreadyExist(value) {
+		return this.state.items.find(item => item.values[0] === value);
```

Comment:
discussed with @gvaznunes, it must be possible to add the same value with different case.